examples/basic_functionality/START_HERE.ipynb

```diff
@@ -0,0 +1,353 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Basic embedding retrieval with Chroma\n",
+    "\n",
+    "This notebook demonstrates the most basic use of Chroma to store and retrieve information using embeddings. This core building block is at the heart of many powerful AI applications.\n",
+    "\n",
+    "## What are embeddings?\n",
+    "\n",
+    "Embeddings are the A.I-native way to represent any kind of data, making them the perfect fit for working with all kinds of A.I-powered tools and algorithms. They can represent text, images, and soon audio and video.\n",
+    "\n",
+    "To create an embedding, data is fed into an embedding model, which outputs vectors of numbers. The model is trained in such a way that 'similar' data, e.g. text with similar meanings, or images with similar content, will produce vectors which are nearer to one another, than those which are dissimilar.\n",
+    "\n",
+    "## Embeddings and retrieval\n",
+    "\n",
+    "We can use the similarity property of embeddings to search for and retrieve information. For example, we can find documents relevant to a particular topic, or images similar to a given image. Rather than searching for keywords or tags, we can search by finding data with similar semantic meaning.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
```

Comment:
can we use pip install -q to quiet pip output and make NB more readable

Reply:
the people must see the install output. they must know what the computer is doing for them. thank you, computer.

Reply:
Lol, in that case...can we cheat and install it via quiet but save it as verbose?